data-extraction/data-import/src/main/java/com/wipro/ats/bdre/dataimport/HDFSImport.java

```diff
@@ -66,7 +67,7 @@ public HDFSImport(Properties commonProperties, String[] columns) {
                 cols[i] = columns[i];
         }
     }
-
+    @Override
     public int run(String[] param) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "run" is 23 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)